platforms/vmware/variables.tf

```diff
@@ -114,6 +114,19 @@ variable "tectonic_vmware_etcd_clusters" {
 EOF
 }
 
+variable "tectonic_vmware_etcd_resource_pool" {
+  type = "map"
+
+  description = <<EOF
+  terraform map of etcd node(s) vSphere Resource Pools, example:
+  tectonic_vmware_etcd_clusters = {
```

Comment:
Please rename all description variables as resource_pool, not clusters e.g. 

tectonic_vmware_etcd_clusters -> tectonic_vmware_etcd_resource_pool.

The same applies for the descriptions for the master and worker resource pools